pkgs/by-name/ka/kazv/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  boost,
+  cmake,
+  cmark,
+  cryptopp,
+  extra-cmake-modules,
+  immer,
+  kdePackages,
+  lager,
+  libkazv,
+  nlohmann_json,
+  olm,
```

Comment:
Please be aware that as of https://github.com/NixOS/nixpkgs/pull/334638 `olm` is marked as insecure and both packages won't be build, unless you set `permittedInsecurePackages` or `export NIXPKGS_ALLOW_INSECURE=1`

Reply:
I know, and that's fine by me. By the way, @tusooa (kazv dev) is spearheading the ecosystem-wide migration of c++ clients to vodozemac-bindings https://iron.lily-is.land/project/view/10/ :)

Reply:
the next release will be using vodozemac and no longer depend on olm.